docs/source/remote_data.rst

```diff
@@ -79,7 +79,7 @@ Historical dividends from Yahoo! Finance.
 .. _remote_data.yahoo_quotes:
```

Comment:
this got changed to 100755 for some reason, can you switch back

something like

``git update-index --chmod -x <file>`` should do it